templates/base.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    {% block title %}
+        <title>Taxi Service</title>
+    {% endblock %}
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'taxi/css/styles.css' %}">
+</head>
+
+
+<body>
+
+    {% block topbar %}
+        {% include "includes/topbar.html" %}
+    {% endblock %}
+
+    <h1>WELCOME !!!</h1>
+
+    {% block content %}
+        {% include "includes/content.html" %}
```

Comment:
`base.html` is a base template for the whole site, but you included the specific taxi app information here. Remove.